src/classes/storage.ts

```diff
@@ -33,4 +64,70 @@ export class Storage {
    * @returns {void}
    */
   public deleteStopCluster(): void {}
+
+  /**
+   * @description Adding StopCluster to the ground places list.
+   * @param {GroundPlaceGenerated} stopCluster - The stop cluster to add to the ground places list.
+   * @returns {void}
+   */
+  public addStopClusterToGroundPlacesList({
+    id: stopClusterGpuid,
+    name,
+    latitude,
+    longitude,
+    countryCode,
+    type,
+  }: GroundPlaceGenerated): void {
+    // Create the StopCluster
+    const newStopCluster = {
+      [stopClusterGpuid]: { name, latitude, longitude, country_code: countryCode, type, childs: [] },
+    };
+
+    // TODO: Call checkValidity on the groundPlacesList to check if all rules are respected before adding the StopCluster
+
+    // Append the StopCluster to the ground places list.
+    this.groundPlacesList = {
```

Comment:
Wow this is WEIRD

Reply:
Removed, better using separate methods like `create`, `remove`, `move` that are driven by the `groundPlacesDiff` object